backend/src/session/__init__.py

```diff
@@ -0,0 +1,15 @@
+from .redis_session_middleware import RedisSessionMiddleware
+from .chat import Message, clear_session_chat, get_session_chat, update_session_chat
+from .cypher_query import CypherQuery, clear_session_cypher_query, get_session_cypher_query, update_session_cypher_query
+
+__all__ = [
+    "RedisSessionMiddleware",
+    "Message",
+    "clear_session_chat",
```

Comment:
I don't think we ever use the clear session methods, do we want to implement these somewhere as currently the history is carried over between sessions